packages/google_maps_flutter/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -300,6 +309,18 @@ - (void)onInfoWindowTap:(NSString*)markerId {
     [_methodChannel invokeMethod:@"infoWindow#onTap" arguments:@{@"markerId" : markerId}];
   }
 }
+- (void)showMarkerInfoWindow:(NSString*)markerId {
+  FLTGoogleMapMarkerController* controller = _markerIdToController[markerId];
```

Comment:
`controller` can be nill.  Although objc won't crash `[controller showInfoWIndow];` , we should still have a logic somewhere to tell `dart` that the markerId is invalid.